csvcubed/csvcubed/readers/skoscodelistreader.py

```diff
@@ -52,10 +58,18 @@ def extract_code_list_concept_scheme_info(
             + f"Expected 1, found {len(variables_in_about_url)}"
         )
 
-    if "notation" not in variables_in_about_url:
-        raise ValueError(
-            "Unexpected variable found in aboutUrl template. Expected 'notation', "
-            + f"found '{variables_in_about_url.pop()}'"
+    variable_name_in_about_url = first(variables_in_about_url)
+    assert variable_name_in_about_url is not None
```

Comment:
I suggest we throw an exception here instead of asserting.

Reply:
I'd normally agree with you, but that assert statement will always be false and is only there to make it clear to pyright that the type is `str` rather than `Optional[str]`.